specs/commandline/verify.md

```diff
@@ -37,9 +37,11 @@ Usage:
 Flags:
   -d,  --debug                       debug mode
   -h,  --help                        help for verify
+       --oci-layout                  [Experimental] verify the artifact stored in OCI image layout
   -p,  --password string             password for registry operations (default to $NOTATION_PASSWORD if not specified)
        --plain-http                  registry access via plain HTTP
        --plugin-config stringArray   {key}={value} pairs that are passed as it is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values
+       --scope string                [Experimental] set trust policy scope for artifact verification, required to be used with flag "--oci-layout"
```

Comment:
Don't know how to phrase it better but we should mention that `scope` can only be used when `--oci-layout` is specified.